src/root/views/register.js

```diff
@@ -412,6 +462,9 @@ class Register extends React.Component {
                   value={this.state.data.username}
                   onChange={this.onFieldChange.bind(this, 'username')}
                 >
+                  <p className='text-italic'>
+                    This will be your username to log into the platform
+                  </p>
```

Comment:
Looks like you've missed the translation here

Reply:
Fix via: https://github.com/IFRCGo/go-frontend/pull/1377/commits/fb55d819b67f460aa38a86183c0d5bcf32d03e0b